package.json

```diff
@@ -19,5 +20,8 @@
   "devDependencies": {
     "lerna": "^6.4.1",
     "@jupyterlab/builder": "^3.1.0"
+  },
+  "nx": {
+    "includedScripts": []
   }
```

Comment:
This configuration that tells Nx to not include root package scripts as Nx targets is required to stop root package scripts from causing an infinite loop. For example, `yarn build` on root calls `lerna run build`, which calls `yarn build` on root, etc. Now, `lerna run build` does not run on the root project because no targets are defined for that project.